src/Agent.Worker/Release/ContainerProvider/Helpers/VssConnectionFactory.cs

```diff
@@ -55,16 +55,16 @@ public static class VssConnectionFactory
             return connection;
         }
 
-        private static VssClientCredentials GetCredentials(String accessToken)
+        private static VssClientCredentials GetCredentials(string accessToken)
         {
             VssClientCredentials cred;
             if (string.IsNullOrEmpty(accessToken))
             {
-                cred = new VssClientCredentials(new VssAadCredential());
+                cred = new VssClientCredentials(null, new VssAadCredential());
```

Comment:
This could be written differently: 
`cred = new VssClientCredentials(federatedCredential: new VssAadCredential());`. 
I think it's more readable

Reply:
Fixed, thanks!